charts/incubator/alist/values.yaml

```diff
@@ -0,0 +1,18 @@
+image:
+  repository: tccr.io/truecharts/alist
+  pullPolicy: IfNotPresent
+  tag: latest
```

Comment:
Same feedback as for the container repo also applies here:
use version tags and make sure they are digest pinned

Reply:
OK